cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -57,6 +57,7 @@ module Cardano.Api.IPC (
     QueryInEra(..),
     QueryInShelleyBasedEra(..),
     queryNodeLocalState,
+    queryNodeLocalState_,
```

Comment:
We export `queryNodeLocalState_` so it can be used, but we do not use them yet.

All functions that end with `_` will by convention be support `oops` style error handling.  The function of the same name, but with the ending underscore is a non-`oops` wrapper that is kept for backwards compatibility or provide a way for developers to use our library without needing to deal with `oops`.